plugin.cpp

```diff
@@ -35,29 +35,102 @@ static const char *default_config = QUOTE({
 	"brokers": {
 		"description": "The bootstrap broker list to retrieve full Kafka brokers",
 		"type": "string",
-		"order": "1",
+		"order": "9",
 		"displayName": "Bootstrap Brokers", 
 		"default": "localhost:9092,kafka.local:9092"
 		},
 	"topic": {
 		"description": "The topic to send reading data on",
-		"order": "2",
+		"order": "10",
 		"displayName": "Kafka Topic",
-		"type": "string", "default": "Fledge"
+		"type": "string", "default": "FogLAMP"
 		},
 	"json": {
 		"description": "Send as JSON objects or as strings",
 		"type": "enumeration",
 		"default": "Strings",
-		"order": "3",
+		"order": "11",
 		"displayName": "Send JSON",
 		"options" : ["Objects","Strings"]
 		},
+	"KafkaSecurityProtocol": {
+		"description": "Security protocol to be used to connect with kafka broker",
+		"type": "enumeration",
+		"default": "PLAINTEXT",
+		"order": "12",
+		"group": "Authentication",
```

Comment:
probably group should be `Kafka Authentication` and remove Kafka from display name for items under this group.